app/src/main/java/com/hedvig/app/feature/offer/OfferRepository.kt

```diff
@@ -61,18 +51,47 @@ class OfferRepository(
         }
     }
 
+    suspend fun queryAndEmitOffer(quoteCartId: CreateQuoteCartUseCase.QuoteCartId?, quoteIds: List<String>) {
```

Comment:
Exactly, with this solution we need to call `queryAndEmitOffer` after every mutation done to the cart. This will cause `offerFlow` to re-emit. 

When we are calling .first() on the flow, we assume that there already is a value. Should we call `queryAndEmitOffer` in these places as well do you think? Or can we solve it another way?